.vscodeignore

```diff
@@ -1,5 +1,22 @@
-.vscode
+# The node modules are bundled
 node_modules
-out/
-tsconfig.json
-webpack.config.js
\ No newline at end of file
+
+# The VSCode Extension's dev artifacts
+.idea/**
+.pipeline/**
+.vscode/**
+.vscode-test/**
+.nyc_output/**
+.circleci/**
+.github/**
+.reuse/**
```

Comment:
I'm not sure about the `.reuse` since it's licensing information. Maybe we should keep it. Please consult with Michal or Shachar.